tests/data/remove_with_brackets.py

```diff
@@ -0,0 +1,79 @@
+with (open("bla.txt")):
```

Comment:
Could you add test cases with multiple levels of parentheses here?

(And probably also foor `for`, in `remove_for_brackets.py`)

Reply:
I've added the multi-paren examples for `with` in that last commit 👍 

FYI I've noted as well that only the context manager can have multiple paren levels so that's why these examples below aren't included in those tests 😄 
```python
# This is valid syntax
with (open("bla.txt") as f):
    pass

# This is bad syntax
with ((open("bla.txt") as f)):
    pass

# This is valid syntax
with (open("bla.txt") as f, open("bla.txt") as f):
    pass

# This is bad syntax
with ((open("bla.txt") as f, open("bla.txt") as f)):
    pass
```

Reply:
will add the `for` ones as well 👍

Reply:
Got the `for` examples in as well